### DISCUSSION TITLE
usb Rubber ducky

### DISCUSSION THREAD

**Discussion by gooseisgoose** [3kh0/ext-remover#942] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there anything on here that i could do w a usb rubber ducky

**Comment by AshtonDavies** [comment]
I really could be wrong, but there's probably nothing. Speaking in the subject of this repository, that cannot bypass things on a school Chromebook I am pretty sure. If there is a powerful command/action to screw with something, I feel like it would be blocked by the operating system. Just my thought on it, never had one of those, but it's a cool device and that's a really good question.

**Reply by gooseisgoose** [reply]
its not blocked. when i plug it in to run a payload it works, my school district is weird you can install nmap but you can't go to a game website. thanks anyway i ask a lot of question's that are not really related to anything here.

**Reply by AshtonDavies** [reply]
> its not blocked. when i plug it in to run a payload it works. thanks anyway i ask a lot of question's that are not really related to anything here. 

I was referring to stuff in here because that is what you put in the title, but anyway, it's nice that the payload works.

**Reply by gooseisgoose** [reply]
ok sorry 
> I was referring to stuff in here

**Reply by gooseisgoose** [reply]
i could mabey install some sort of brute forcer?

**Reply by AshtonDavies** [reply]
> i could mabey install some sort of brute forcer?

I think that is a genius idea.

**Comment by 3kh0** [comment, marked as answer]
In theory, it should work on all devices, including Chromebooks. In terms of doing an actual exploit? Most likely not. But you can, however, plug it into your friend's computers and make it look up big oily men. Just a thought.

**Reply by AshtonDavies** [reply]
I thought most devices had protection from USB injection. Especially for Chromebooks, you'd expect these watered-down laptops to only read filesystems and basic inputs.

Edit: Okay, it does do basic inputs which can go undetected to malware scanners, so, it's kinda like an automated Makey Makey, but much more useful and better.

**Comment by typicallol** [comment]
> its not blocked. when i plug it in to run a payload it works, my school district is weird you can install nmap but you can't go to a game website. thanks anyway i ask a lot of question's that are not really related to anything here.

yo when i try to rickroll my self it wont work gimme a tutorial plz my payloads dont work

**Reply by gooseisgoose** [reply]
what do you mean it wont work. i could right you one if you want

**Reply by gooseisgoose** [reply]
have you tried using payloads from [here](https://github.com/hak5/usbrubberducky-payloads/tree/master/payloads/library) and you use payload studio right.

**Comment by lolpc100** [comment]
but use a flipper zero

**Reply by gooseisgoose** [reply]
how good is the bad usb on flipper. does it work like the usb rubber ducky, i've never used a flipper.
